keel-sql/src/main/resources/db/changelog/20190227-initial-schema.yml

```diff
@@ -0,0 +1,92 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-resource-table
+      author: fletch
+      changes:
+        - createTable:
+            tableName: resource
+            columns:
+              - column:
+                  name: uid
+                  type: char(26)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: api_version
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: kind
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    unique: true
+                    nullable: false
+              - column:
+                  name: resource_version
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: spec
+                  type: longtext
+                  constraints:
+                    nullable: false
+        - modifySql:
+            dbms: mysql
+            append:
+              value: " engine innodb"
+      rollback:
+        - dropTable:
+            tableName: resources
+  - changeSet:
+      id: create-resource-state-table
+      author: fletch
+      changes:
+        - createTable:
+            tableName: resource_state
```

Comment:
Is this table going to remain update only? If you plan on deleting rows from here in the future, doing so via a `ulid` based primary key column is preferable. (Deleting rows other than via a unique key can effectively block inserts)

Reply:
I’d like to update it to use the uid. Was just doing it this way as the repository interface was already there.

Reply:
But yeah, it’s an append only table